downstream/modules/aap-hardening/ref-complex-patching-scenarios.adoc

```diff
@@ -0,0 +1,20 @@
+[id="ref-complex-patching-scenarios"]
+
+= Complex patching scenarios
+
+In {PlatformNameShort}, multiple automation jobs can be chained together into workflows, which can be used to coordinate multiple steps in a complex patching scenario. 
+
+The following example complex patching scenario demonstrates taking virtual machine snapshots, patching the virtual machines, and creating tickets when an error is encountered in the workflow. 
+
+. Run a project sync to ensure the latest playbooks are available. In parallel, run an inventory sync to make sure the latest list of target hosts is available.
+. Take a snapshot of each target host.
+.. If the snapshot task fails, submit a ticket with the relevant information.
+. Patch each of the target hosts.
+.. If the patching task fails, restore the snapshot and submit a ticket with the relevant information.
+. Delete each snapshot where the patching task was successful.
+
+The following workflow visualization shows how the components of the example complex patching scenario are executed:
```

Comment:
No workflow visualization present?

Reply:
I don't know where this image comes from. I'm meaning to ask in the meeting.